pkg/ottl/README.md

```diff
@@ -106,15 +107,34 @@ Within the grammar Enums are always used as `int64`.  As a result, the Enum's sy
 
 When defining a function that will be used as an Invocation by the OTTL, if the function needs to take an Enum then the function must use the `Enum` type for that argument, not an `int64`.
 
-### Expressions
+#### Expressions
```

Comment:
I thought about that too, but was getting tripped up by the potential use case of `"string" + "string"`.  If we determine that operations will only be used to conduct math, then I'm good changing the name.

Reply:
I am thinking it may be best to restrict string concatenation to the `Concat` function to keep things simple. That way the string conversion behaviors are clear and explicit (`1 + 1 + "1" = ?`) and there's a single way to do things, which I think has value. I am open to other opinions on this, though.

Reply:
Since we already have concat, I'm good with that.  In the future if we needed to undoing a restriction to numbers it wouldn't even be a breaking change.